### DISCUSSION TITLE
Investigate streaming use cases and how they might be supported in Falcon

### DISCUSSION THREAD

**Discussion by kgriffs** [falconry/falcon#2034] [Ideas]
For example: http://blog.miguelgrinberg.com/post/video-streaming-with-flask

**Comment by vytas7** [comment]
Moving this to a discussion since the scope is quite broad, and it's probably better suited for a third party component or a blog post.
As of 3.1, static routes will support streaming simple HTML5 videos with `Range` requests (https://github.com/falconry/falcon/issues/1857). We also have some recipes how to stream content [on-the-fly](https://falcon.readthedocs.io/en/stable/user/recipes/output-csv.html#streaming-large-csv-files-on-the-fly).

**Comment by vytas7** [comment]
Maybe we need an article or a docs chapter along the lines of uWSGI's [Fun with Perl, Eyetoy and RaspberryPi](https://uwsgi-docs.readthedocs.io/en/latest/articles/FunWithPerlEyetoyRaspberrypi.html). :grin:
